SmartPlaces.Facilities/lib/IngestionManager/src/IngestionManager.csproj

```diff
@@ -29,6 +29,7 @@
 		<PackageReference Include="Microsoft.Extensions.Http" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0" />
+		<PackageReference Include="Microsoft.SmartPlaces.Facilities.OntologyMapper" Version="0.6.0-preview" />
```

Comment:
Its not letting me comment in a relevant spot, but shouldn't the version be revved?